dffml/operation/archive.py

```diff
@@ -41,8 +42,16 @@ async def make_zip_archive(
         Path to the output zip file
     """
     with zipfile.ZipFile(output_file_path, "w") as zip:
-        for file in pathlib.Path(input_directory_path).rglob("*"):
-            zip.write(file, file.name)
+        # TODO: Make a clean implementation of this with pathlib
+        for root, _, files in os.walk(input_directory_path):
```

Comment:
Let's look to https://github.com/python/cpython/blob/7ef0673de48958bc3a75af5f152564bd2dffa8dd/Lib/zipfile.py#L2366-L2424 as a reference

Reply:
Shall I make an enhancement issue for this ? 
As it would take time to verify the changes and I need to hit the deadline. 
It works fine currently , just the implementation is not ideal I'd say, 
going for a better implementation would require extra time and testing, 
so I will try to do it as soon as possible after the deadline ......
does that sound good to you ?

Reply:
Ok, looking into it , would push a similar solution asap .....

Reply:
by the way can we create an enhancement issue for this and come back to it later ?

Reply:
Done, let me know if any other changes are required .....